contentcuration/contentcuration/frontend/shared/views/channel/ChannelDetailsModal.vue

```diff
@@ -187,14 +182,24 @@
 
 <style lang="scss" scoped>
 
-  .v-toolbar__title {
-    font-weight: bold;
+  .download-button-container {
+    display: flex;
+    justify-content: flex-end;
+    margin-bottom: 24px;
   }
 
-  .draft-header {
-    padding-right: 10px;
-    font-style: italic;
-    color: gray;
+  [dir='rtl'] .download-button-container {
+    justify-content: flex-start;
+  }
+
+  @media (max-width: 600px) {
```

Comment:
We avoid CSS media queries in favor of our layout system - mostly for consistency. Can you use `useKResponsiveWindow`?

- https://design-system.learningequality.org/layout#responsiveness
- https://design-system.learningequality.org/usekresponsivewindow/